src/main/webapp/creatives/maps/Logger.js

```diff
@@ -0,0 +1,53 @@
+
+import {generateSessionId} from './utils.js';
+
+/** A list of server routes that keep track of each interaction when fetched. */
+const URL = {
+  INITIALIZE: "initialize",
+  FIND_NEAREST_LOCATION: "find-location",
+  GRANT_LOCATION_DATA: "grant-location",
+  MAP_INTERACT: "map-interact",
+  SKIP_TO_CONTENT: "skip-to-content",
+  RETURN_TO_AD: "return-to-ad"
+}
+
+class Logger {
```

Comment:
imo you can be a bit more descriptive here. `InteractionLogger` or `UserActivityLogger` or something like that

Reply:
yes^ Logger is not any more descriptive than "User"